### ISSUE TITLE
Docker image not up-to-date since there's no 'apt-get upgrade' command

### ISSUE THREAD

**Issue by mykaul** [scylladb/scylla-cqlsh#59]
See https://github.com/scylladb/scylla-cqlsh/blob/426fa0eaa72b7d46115188b1af9cdd983b923241/pylib/Dockerfile.ubuntu.py3 - similar/identical issue as https://github.com/scylladb/scylladb/issues/16222

CC @yaronkaikov

**Comment by mykaul** [comment]
Perhaps https://github.com/scylladb/scylla-cqlsh/blob/426fa0eaa72b7d46115188b1af9cdd983b923241/Dockerfile#L11 is also relevant here.

**Comment by fruch** [comment]
> Perhaps
> 
> https://github.com/scylladb/scylla-cqlsh/blob/426fa0eaa72b7d46115188b1af9cdd983b923241/Dockerfile#L11
> 
> is also relevant here.

yes this is the correct one, the first one you mentioned is copied from cassandra code, and was used just for testing, not for publishing

the Dockerfile in the root folder is.

but it's still the same we are using the official python images, which get updates once there a new release.
we are using the slim version to make sure we don't have any unneeded things.

but we should scan this image, and see what's there, not blindly upgrade, it can bite us as well.

**Comment by mykaul** [comment]
Scanning latest image, let's see. The thing is, they don't update often in and in between the underlying image - but let me complete the scanning and we'll see.
